lms/djangoapps/courseware/views/views.py

```diff
@@ -402,10 +439,6 @@ def get_last_accessed_courseware(course, request, user):
 
     course_key = CourseKey.from_string(course_id)
 
-    # If the unified course experience is enabled, redirect to the "Course" tab
-    if UNIFIED_COURSE_TAB_FLAG.is_enabled(course_key):
-        return redirect(reverse(course_home_url_name(course_key), args=[course_id]))
-
```

Comment:
I think you should keep this code around since this will handle redirects from `courses/<course id>/info` to the course home.

Reply:
Added it back again.